apiary.apib

```diff
@@ -25,10 +26,20 @@ The majority of errors along to the error code provide additional information in
 }
 ```
 
-## Group Token
-
-Some requests require an access token for authorization.
+## Group Token [Token]
 
+Some requests like account management or accessing to restricted data fields require an access token for authorization. The access token is a base64-encoded JSON file of following format:
```

Comment:
We should also add that we are using a deterministic way to serialise the JSON into a string. This should probably be explained in detail in a separate section, with a link here.